content/04.results.md

```diff
@@ -34,3 +34,15 @@ A) Counts of nonsynonymous mutations, colored by tumor type (as defined in `inte
 B) Co-occurrence and mutual exclusivity of nonsynonymous mutations between genes.
 The co-occurrence score is defined as $I(-\log_{10}(P))$ where $P$ is defined by Fisher's exact test and $I$ is 1 when mutations co-occur more often than expected and -1 when exclusivity is more common.
 ](https://github.com/AlexsLemonade/OpenPBTA-analysis/raw/master/figures/pngs/mutation_cooccurrence_figure.png){#fig:occurrence-plots}
+
+### Transcriptomic landscape
+
+This section will discuss the overall structure of the transcriptome data, pathway analysis, and immune deconvolution.
+A dimension reduction plot, a heatmap of GSVA scores [@doi:10.1186/1471-2105-14-7], and a heatmap of xCell fraction values [@doi:10.1186/s13059-017-1349-1] are shown in Figure {@fig:transcriptomic-overview}.
+
+![Transcriptomic overview (stranded data only)
+A) First two dimensions from Uniform Manifold Approximation and Projection (UMAP).
```

Comment:
I don't love exactly what I wrote here, but it should say something about what the UMAP was performed on, I think.
```suggestion
A) First two dimensions from Uniform Manifold Approximation and Projection (UMAP) of sample transcriptome data.
```